source/manual/add-support-new-language.html.md

```diff
@@ -0,0 +1,96 @@
+
+- [Frontend](/repos/frontend.html)
+- [Government-frontend](/repos/government-frontend.html)
+- [Govuk-publishing-components](/repos/govuk_publishing_components.html)
+- [Govuk-app-config](/repos/govuk-app-config.html)
+
+Developers must export a csv containing all internationalised strings from frontend, government-frontend and govuk-publishing-components to be sent to the translator. Documentation on how to do this is available in [Rails translation manager](/repos/rails_translation_manager/translating-locale-files.html).
+
+Check if the new language is already supported by the Rails framework by looking for your language tag in the [rails locale folder][]. If it's not supported, translations will also be needed for govuk-app-config. The internationalised strings that Rails requires contain a lot of interpolation, which has proved challenging for translators to understand. There is a [template csv file][] that contains instructions for translators which can be used for the govuk-app-config translations.
+
+[rails locale folder]:[https://github.com/svenfuchs/rails-i18n/tree/master/rails/locale]
+[govuk-app-config Readme]:[https://github.com/alphagov/govuk_app_config?tab=readme-ov-file#internationalisation]
+[template csv file]:[https://docs.google.com/spreadsheets/d/12lhWVXz30l3c8WTakK_EIxJLUBoLJO80lekvq-treww/edit?usp=sharing]
+
+Once translations have been obtained from the translators, publishing and rendering applications can be updated via the following steps:
+
+> Do not complete steps 3, 4 and 5 before steps 1 and 2 have been completed. This can cause failures in our CI/CD when the frontend rendering applications are asked to render content for all the locales publishable by whitehall.
+
+### 1. Add support for your new language to the frontend rendering applications
+
+At the time of writing, Whitehall content is rendered by either Frontend or Government Frontend. So both applications must be updated.
+
+[Example PR in government](https://github.com/alphagov/government-frontend/pull/1382)
+
+1. Add the new language tag to `config/application.rb` and `config/locales/en.yml` in alphabetical order
+2. From within frontend, or government frontend, create a folder `tmp/locale_file`
+3. Add your csv from the translator to this folder. The name of the csv should be in the format: `<new_locale>.csv`, eg `ky.csv`
+4. Run the following commands to import your csv, and create a new locale file.
+
+    ```bash
+    $ rake translation:import_all
+    ```
+
+3. In `config/locales/<new_locale>.yml` add the language translation under the `language_names` key.
+
+    For example:
+
+     ```yaml
+     it:
+       language_names:
+         it: italiano
+     ```
+
+### 2. Add support for your new language to govuk-publishing-components
+
+Follow steps 2, 3 and 4 from the [previous section](#Add-support-for-your-new-language-to-the-frontend-rendering-applications) from within the govuk-publishing-components repo.
```

Comment:
This won't work otherwise:

```suggestion
Follow steps 2, 3 and 4 from the [previous section](#1-add-support-for-your-new-language-to-the-frontend-rendering-applications) from within the govuk-publishing-components repo.
```